src/components/icon-button/icon-button.tsx

```diff
@@ -63,11 +58,11 @@ export class IconButton {
         }
     }
 
-    private removeFocusedStyleOnClick() {
+    private removeFocusedStyleOnClick = () => {
```

Comment:
Put this change in other commit and the type of change is not `docs`. It should be `refactor`

Reply:
this seems not to be from the examples, so this was done by mistake. :/ I don't want to change anything in the components' code in this PR

Reply:
ok then you remove this change, right?

Reply:
I'll do

Reply:
pushed a fixup